app/views/users/_metadata.html.erb

```diff
@@ -1,7 +1,8 @@
-<% if FeatureFlag.enabled?(:profile_admin) %>
+<% if true # FeatureFlag.enabled?(:profile_admin) %>
   <% if (header_fields = @profile.ui_attributes_for(area: :header)).present? %>
     <div class="profile-header__bottom fs-base">
       <% header_fields.each do |title, value| %>
+        <% next if Profile.special_attributes.include?(title) %>
```

Comment:
I think I understand the idea behind this BUT does special for DEV automatically imply special for other Forems?

Reply:
Yeah, this essentially makes these reserved words for the time being. This doesn't prevent further generalization, but if things like summary, location, etc. are special I think that's okay... Especially brand_color1 etc. That needs to be special.

I think we can clarify this in the interface.

Reply:
I think we're hitting on something interesting here: maybe `brand_color1` and `brand_color2` aren't really profile fields (even though they are set by each user), but some sort of setting instead? Given that we're still working on profile generalization, now would be the perfect time to also introduce other models to move fields to instead of just moving everything from `User` to a `ProfileField`.